MekHQ/src/mekhq/gui/view/MissionViewPanel.java

```diff
@@ -9,15 +9,17 @@
 import java.awt.*;
```

Comment:
Ended up mixed tabs and spaces.

Reply:
Just switched over to intelliJ and probably need to futz with some settings

Reply:
I need to get us an editorconfig so they'll all just work.

Reply:
Tabs should all be gone now.